web/src/App.jsx

```
@@ -2076,6 +2077,15 @@ const Sidebar = memo(({
                         Premium Required
                       </div>
                     )}
+                    {isPremium && (
+                      <button
+                        className="secondary-btn"
+                        style={{ marginTop: '12px', width: '100%', fontSize: '0.85em' }}
+                        onClick={requestPhoneSync}
+                      >
+                        Request Phone Sync
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)